### ISSUE TITLE
Question about manually bumping commit.

### ISSUE THREAD

**Issue by GurdipS5** [dotnet/Nerdbank.GitVersioning#1078]
Hi,
I am using NBGV in an NPM project.

I am a bit confused, as I thought NBGV could automatically increase the minor version per build? I think this is how the Nuget package for .NET Core works.

So am I using this correctly to amend the version no manually before each commit?

Thanks!

**Comment by AArnott** [comment]
NB.GV (and the nbgv CLI tool) calculates the version based on git history. It does _not_ increment the version for every single build. If you add a commit, the computed version is incremented automatically. No need to touch version.json in every commit for this to happen.

**Comment by GurdipS5** [comment]
That makes sense. However, I have my commit but there's no change in the version.json file?

**Comment by AArnott** [comment]
Committing doesn't change the version.json file. But nbgv or the NB.GV nuget package will calculate a higher version for each successive commit anyway.

**Comment by GurdipS5** [comment]
Yeah that makes sense.I guess that leaves my last question which is to get the higher version number, is there a command? I as looking at get-version etc.

**Comment by AArnott** [comment]
You get a higher version number with every commit. And if you want a number in the version to increase that is actually specified in the version.json file, you need to update the version.json file and commit the change.
